src/main.rs

```diff
@@ -34,9 +36,27 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let (layer, io) = SocketIo::builder().with_state(pool).build_layer();
 
     io.ns("/", on_connect);
-    let app = Router::new()
-        .layer(layer)
-        .layer(CorsLayer::very_permissive());
+    // Get the allowed origins from the .env file
+    let allowed_origins = env::var("ALLOWED_ORIGINS").expect("ALLOWED_ORIGINS must be set");
```

Comment:
Instead of panicking here, we can default it to `localhost:5173` (the dev frontend) and log a trace about the same `trace::warning!("ALLOWED_ORIGINS not found in .env, defaulting to....");`